app/reducers/index.js

```diff
@@ -1,5 +1,19 @@
-const stockTicker = (state = {}, action) => {
-    switch (action.type) {
+import { ADD_NEW_PRICE_TICKER } from './actions';
+
+const initialState = {
+    ticker: null,
+    oldPrice: 0,
+};
+
+const stockTicker = (state = initialState, {type, payload}) => {
+    switch (type) {
+        case ADD_NEW_PRICE_TICKER : {
+            const newState = JSON.parse(JSON.stringify(state));
```

Comment:
better to use `...` (destructuring) es6 syntax to create new object without parent (if exists) prototype